stages/org.osbuild.oscap.remediation

```diff
@@ -6,10 +6,12 @@ The OpenSCAP scanner can be run on the image tree and the remediation can be car
 out during build time. The stage takes the OpenSCAP config as input and then runs the
 the utility in chroot to scan and remediate [1] the tree during image at build time.
 
-The stage generates an html report and xml results file both saved to the `/openscap_data`
-directory.
+The stage generates an html report and xml results file both saved to the `data_dir`
+directory. It defaults to `/root` if not configured.
```

Comment:
Please note the default in the schema below. In addition, is there some good reason to change the default from `/openscap_data` ?

Reply:
I believe it never actually was `/openscap_data` by default, the doc lied. Also, it feels like littering.

Reply:
Yeah, the path was not explicitly used by the stage before. Since I'm not really familiar with the `oscap` tool, I simply assumed that the `/openscap_data` is the path used by the tool in case no path is specified. If this is not the case, then the change looks good (I mean, it looked good in any case, I was mostly curious).